lib/internal/crypto/webidl.js

```diff
@@ -180,7 +180,10 @@ converters.object = (V, opts) => {
 };
 
 function isNonSharedArrayBuffer(V) {
-  return ObjectPrototypeIsPrototypeOf(ArrayBufferPrototype, V);
+  // SharedArrayBuffers can be disabled with --no-harmony-sharedarraybuffer.
+  if (SharedArrayBuffer !== undefined)
+    return ObjectPrototypeIsPrototypeOf(ArrayBufferPrototype, V);
+  return false;
```

Comment:
Are you not fixing the wrong function?

The error is on `isSharedArrayBuffer`.

Reply:
yes, patch application mistake. corrected.